kempner_hpc_handbook/development_and_runtime_envs/using_vscode_for_remote_development.md

```diff
@@ -26,6 +26,11 @@ Host cannon
   HostName holylogin01.rc.fas.harvard.edu
   User <username>
 ```
+
+```{warning}
+Please rotate among different login nodes to avoid an unbalanced load on specific nodes. Create multiple connections and change the login node name (e.g., from `holylogin01` to `holylogin02`, `holylogin03`, or `holylogin04`) and connect to a different login node every time.
+```
+
```

Comment:
Do you mean creating cannon01, cannon02, and so on. If it is the case, it would be good to add an example, too.

Reply:
I'm going to add it now

Reply:
@Naeemkh , added an example for the first occurrence which should be enough to cover the 2nd use case.